api/src/resource-extraction.js

```diff
@@ -7,35 +7,24 @@ Therefore, we extract all cacheable resources into a folder and serve them as pu
 const
   fs = require('fs'),
   path = require('path'),
-  { env } = require('process'),
   db = require('./db'),
+  environment = require('./environment'),
   logger = require('./logger'),
-  isAttachmentCacheable = name => name === 'manifest.json' || !!name.match(/(?:audio|css|fonts|templates|img|js|xslt)\/.*/);
+  isAttachmentExtractable = name => name === 'manifest.json' || !!name.match(/(?:audio|css|fonts|default-docs|templates|img|js|xslt)\/.*/);
```

Comment:
Do you think this regex would be clearer as a loop of startswith calls?

Also while you're here, I think the `xslt` bit can be removed now that we're generating xforms on the server.

Reply:
Seems we just want to extract everything except `compiled.json` ...  is such a complex filter necessary?

Reply:
Also I think there are the api and sentinel zip files we don't want to extract. There might be others.

We could switch to a deny list but we're more likely to miss something getting extracted accidentally than something not being extracted when it's needed.

The other way would be to attach all the resources we want to extract in a common folder so we just extract one folder and we're done. But that's probably too much effort and risk to do so late in the release cycle. Maybe raise a new issue?

Reply:
And we don't extract translations. Things seem sufficiently complex either way. I think your original suggestion is a reasonable path forward. Unsure if it a clear improvement to have the more complex folder structure at build time.